util/log/field.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2015 The Cockroach Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+// implied. See the License for the specific language governing
+// permissions and limitations under the License. See the AUTHORS file
+// for names of contributors.
+//
+// Author: Tobias Schottdorf
+
+package log
+
+// A Field is an integer used to enumerate allowed field names in structured
+// log output.
+type Field int
+
+//go:generate stringer -type Field
+const (
+	ClusterID Field = iota
```

Comment:
I guess I still don't see why we can't just have a single struct with these fields and store that in Context.Value. The struct could have atomic getters and setters. Setting this many details as values seems like an abuse of context.Value.